pkg/osbuild/monitor_test.go

```diff
@@ -14,9 +14,9 @@ import (
 	"github.com/osbuild/images/pkg/osbuild"
 )
 
-const osbuildMonitorLines_curl = `{"message": "source/org.osbuild.curl (org.osbuild.curl): Downloaded https://rpmrepo.osbuild.org/v2/mirror/public/f39/f39-x86_64-fedora-20231109/Packages/k/kpartx-0.9.5-2.fc39.x86_64.rpm\n", "context": {"origin": "org.osbuild", "pipeline": {"name": "source org.osbuild.curl", "id": "598849389c35f93efe2412446f5ca6919434417b9bcea040ea5f9203de81db2c", "stage": {}}, "id": "7355d3857aa5c7b3a0c476c13d4b242a625fe190f2e7796df2335f3a34429db3"}, "progress": {"name": "pipelines/sources", "total": 4, "done": 0}, "timestamp": 1731589338.8252223}
```

Comment:
Removing the control character makes the test less realistic, this is especially problematic when we do multiline test. The other issue is that its no longer trivial to just copy and paste output from osbuild to generate test-cases. I would not do it and instead add a comment with context (like "\x1e" is included because this follows the json-seq rfc or something).

The fact that our jsonseq parser is navie and just strips is the control chars instead of properly checking is actually an issue (not a big one) but if we fix it we would then be in a weird situation because the tests would no longer work as they are not realistic anymore.

Reply:
The problem is, this character can only be escaped in regular quotes, it is not visible which is a good argument against it.